src/posit/connect/content.py

```diff
@@ -189,10 +189,18 @@ def dashboard_url(self) -> str:
     def app_role(self) -> str:
         return self.get("app_role")  # type: ignore
 
+    @property
+    def owner(self) -> dict | None:
+        return self.get("owner")
+
     @property
     def permissions(self) -> Permissions:
         return Permissions(self.config, self.session, content_guid=self.guid)
 
+    @property
+    def tags(self) -> List[dict] | None:
```

Comment:
This one is trickier since even if you've said to include `tags`, it may not be present in the payload. So if you do the same as I suggested with `owner`, you'll always make an extra request when there are no tags. 

One option could be that when `find()` creates the `ContentItem`s, if `tags` is in the `include` argument and `tag not in result`, to add `tag = []` to the item. That way, `tags` is always a list, and if it is `None` in the data object at this point, that means we didn't request `tags` in the search query, so do the extra `GET` to fetch them. Does that make sense?

Reply:
It does, but I'm not wary of getting too fancy with this. This seems like behavior we should modify server side. For the `GET` case, let's modify the behavior to mimic what we "want" the sever to do. I think that is implementing the `include` querystring parameter.

For now, I modified the property to return an empty array by default. This seems logical since the default querystring parameters are set to include tags. If the user were to modify the include options, then they should be aware that tags will be empty.